.github/deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy Connect Telegram Bot
```

Comment:
same as before, you miss workflows folder

Reply:
fixed